optaplanner-core/src/main/java/org/optaplanner/core/impl/localsearch/DefaultLocalSearchPhase.java

```diff
@@ -144,6 +151,57 @@ public void stepEnded(LocalSearchStepScope<Solution_> stepScope) {
         }
     }
 
+    private void collectMetrics(LocalSearchStepScope<Solution_> stepScope) {
```

Comment:
Nothing for construction heuristics?
That may be why I was seeing the empty graphs.

We need to figure out how to make this happen for every phase.

Reply:
A bunch of metrics in the original benchmarker code is set up for local search only (but data collection is unrelated to it; should be able to make it every step)

Reply:
Looking at the class it on, it for the statistics that are local search specific